<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1091</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="gestion_personels">
     <attribute name="title">
      <string>Gestion Personels</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>991</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ajoute_personel">
       <attribute name="title">
        <string>Ajouter Personel</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_Ajouterclient">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>340</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>270</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>190</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>230</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Tel:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Adresse:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Prenom:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>ID Personel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_personel">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>340</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="afficher_personels">
       <attribute name="title">
        <string>Afficher Personels</string>
       </attribute>
       <widget class="QTableView" name="TABPERSONEL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>541</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>605</x>
          <y>60</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>ID Personel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_aff_id">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>50</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>180</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>320</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Trier par ordre alphabetique</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>360</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Trier par ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rechercher_personel">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>100</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_imprimer_personels">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>320</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="gestion_postes">
     <attribute name="title">
      <string>Gestion Postes</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>1031</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ajouter_poste">
       <attribute name="title">
        <string>Ajouter Poste</string>
       </attribute>
       <widget class="QLabel" name="label_">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Fonction</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Nombre d'heures de travail</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Departement</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_fonction">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>60</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_heures">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_departement">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>160</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_salaire">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>210</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajoute_rposte">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>320</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_poste">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>320</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="afficher_poste">
       <attribute name="title">
        <string>Afficher Postes</string>
       </attribute>
       <widget class="QTableView" name="TABPOSTE">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>571</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>360</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Trier par Salaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>100</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Perpetua</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Fonction</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_aff_fonction">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>90</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_poste">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>150</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_imprimer_poste">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>360</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="statistiques">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
